packages/flipper_dashboard/pubspec.yaml

```
@@ -1,148 +1,150 @@
+  demo_ui_components: 1.0.0
+  lottie: ^3.1.2
+  flipper_rw:
+  flipper_mocks: any
+  excel: ^4.0.6
+  supabase_models: any
+  supabase_flutter: ^2.8.0
+  uuid: ^4.5.1
+  fl_chart: ^0.70.2
+  flutter_typeahead: ^5.2.0
+  desktop_drop: ^0.5.0
+  fl_chart: ^0.66.2  # Add this line for chart support
```

YAMLlint (1.35.1)

[error] duplication of key "fl_chart" in mapping

(key-duplicates)